internal/host/host_test.go

```diff
@@ -945,16 +956,82 @@ var _ = Describe("UpdateInventory", func() {
 		common.DeleteTestDB(db, dbName)
 	})
 
+	It("Test populate eligibility", func() {
+		testDisks := make([]*models.Disk, 0)
+		expectedDisks := make([]*models.Disk, 0)
+
+		for _, test := range []struct {
+			agentDecision    bool
+			agentReasons     []string
+			serviceDecision  bool
+			serviceReasons   []string
+			expectedDecision bool
+			expectedReasons  []string
+		}{
+			{false, []string{}, true, []string{}, true, []string{}},
```

Comment:
This is not very readable, please use named fields.

Reply:
Will do... GoLand makes you forget how things look without an IDE
![image](https://user-images.githubusercontent.com/10882062/102754833-38053780-4376-11eb-9c57-0e79ce3405fa.png)